.github/workflows/ros-build.yml

```
@@ -1,56 +1,70 @@
+            use_ros_testing: true
 
     container:
       image: ${{ matrix.docker_image }}
 
     steps:
-      - name: Checkout source code
-        uses: actions/checkout@v3
+      - name: Checkout
+        env:
+          TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:19: Double quote to prevent globbing and word splitting

(shellcheck)